Tests/SwiftStdlibTests/DictionaryExtensionsTests.swift

```diff
@@ -163,4 +163,16 @@ final class DictionaryExtensionsTests: XCTestCase {
             [6: ["Bryant"], 5: ["James"], 4: ["Wade", "John"], 0: ["", ""]])
         XCTAssertEqual(Dictionary(grouping: array3, by: \String.count), [:])
     }
+
+    func testGetByKeys() {
+        let dict = ["James": 100,
+                    "Wade": 200,
+                    "Bryant": 500,
+                    "John": 600,
+                    "Jack": 1000]
+        let res = dict.pick(keys: ["James", "Wade", "Jack"])
+        XCTAssertEqual(res, ["James": 100, "Wade": 200, "Jack": 1000])
+        let noResults = dict.pick(keys: ["Michael", "Joe", "Christian"])
+        XCTAssertTrue(noResults.isEmpty)
```

Comment:
Can you add a `nil` test like:

```swift
["James": 100, "Wade": nil].pick(keys: ["James", "Wade"])
```

Pretty sure it'll work properly, but I can't say for certain without a test

Reply:
In order for that to work you'll probably need to declare `dict` as `[String: Int?]`

Reply:
Yea, true, and then in this case we will have a results like `["James":Optional(100), "Wade": nil ]`